modin/pandas/test/test_groupby.py

```diff
@@ -510,24 +526,85 @@ def maybe_get_columns(df, by):
                 pandas_groupby,
                 lambda df: df.nth(0).sort_values("col1").reset_index(drop=True),
             )
+
+    raising_exceptions = None
+    if col1_category:
```

Comment:
In all other cases there are no exceptions, right? I mean don't we hide the errors by setting raising_exceptions=None?

Reply:
Should be the same answer like in https://github.com/modin-project/modin/pull/7067#discussion_r1522033510